images/base/Dockerfile.arch

```diff
@@ -0,0 +1,23 @@
+FROM archlinux/base
+
+SHELL ["/bin/bash", "-c"]
+
+# Install baseline packages
+RUN pacman --noconfirm -Syu \
+    base \
+    base-devel \
+    git \
+    bash \
+    curl \
+    wget \
+    htop \
+    man \
+    vim \
+    sudo \
+    python3 \
+    openssh
+
+# Add coder user.
+RUN useradd -mUs /bin/bash coder && \
+  echo "coder ALL=(ALL) NOPASSWD:ALL" >> /etc/sudoers.d/nopasswd
```

Comment:
This is somewhat different to how we do it in the enterprise-dev image, I figured it would be better to match the others though? https://github.com/cdr/c/blob/master/.environments/dev/Dockerfile#L60-L61

Reply:
Hmm, yeah I think this is probably ok for the base images – we can adjust later on if needed 👍